do/private/docker_test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+ESULT=0
+TEST_EXECS=( $(find test -type f -name "*_tests") )
+if [ -z ${TEST_EXECS[0]} ]; then
+    echo "No Tests to be found"
+    exit
+fi
+
+echo "Running the following tests: ${TEST_EXECS[@]}"
+
+for EXEC in ${TEST_EXECS[@]}; do
+    echo "Running test '${EXEC}'..."
+    NAME=$(basename ${EXEC} | sed -r -n "s/(.*)_tests/\1/p")
+    # This project currently can not reliably run tests in random order. This should be fixed and --gtest_shuffle enabled
+    ${EXEC} --gtest_filter="-*Gpu*" --gtest_output="xml:test/results/${NAME}.xml" --gtest_color=yes # --gtest_shuffle
```

Comment:
You can probably enable gtest_shuffle here. I had do disable it for path_arm

Reply:
Done.